src/classes/ADDR_Addresses_TEST.cls

```diff
@@ -1579,4 +1694,99 @@ public with sharing class ADDR_Addresses_TEST {
             system.assertEquals(!fSeasonal, addr.Default_Address__c);
         }        
     }   
+
+    // Helpers
+    ////////////
+
+    /*********************************************************************************************************
+    * @description Populates Contact's Mailing Address from the Address
+    * @param c Contact
+    * @param sourceAddress Address
+    * @return Contact
+    **********************************************************************************************************/
+    static Contact populateContactAddress(Contact c, Address__c sourceAddress) {
```

Comment:
We can have void in the method signature  and the contact will be updated outside of the  "populateContactAddress" scope. I wanted to make the call more clear; I was following the rule " if a function is going to transform its input argument the transformation should appear as the return value".  However, I can easily change the method name and the unit test to execute the following two lines instead:
populateContactAddress(c, listAddrT[0]);
update c;